Source/Atomic/UI/UISelectList.h

```diff
@@ -77,7 +77,8 @@ class UISelectList : public UIWidget
     virtual bool OnEvent(const tb::TBWidgetEvent &ev);
 
 private:
-
+    //Returns true if we have drag and drop element on select list widget
+    bool handlingDragAndDrop_;
```

Comment:
Can we make sure to initialize this member variable in  the constructor, even if it isn't strictly necessary?